### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by Najeeba-Qarqin**
Markdown previewer 
>  In this PR I created react app
> Installed react-markdown
> Created markdown component 
> added a default markdown 
> added a responsive design 
(https://wondrous-blini-c571da.netlify.app/)

### CODE REVIEW

**Comment by Najeeba-Qarqin** [comment]
Hi @Hasinarahman!
Please review my project!

**Comment by Najeeba-Qarqin** [comment]
Hi @Hasinarahman!
Please review my project!

**Comment by Hasinarahman** [comment]
@Najeeba-Qarqin 

Excellent work! 💯 
🎉 Great Design Implemented
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Markdown Previewer project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your used the React-Markdown good
✔️Your design is responsive

soo in my point of view that is nothing to say you can mention the teacher for approval.

**Comment by Najeeba-Qarqin** [comment]
Hello @ahzamir
Please review my project!

File: src/Markdown.jsx
Comment (ahzamir) [inline src/Markdown.jsx:1]: - Remove this empty file

File: src/Components/MarkdoqnPreveiwer.jsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+![Octocat De Los Muertos](https://octodex.github.com/images/octocat-de-los-muertos.jpg "Octocat De Los Muertos")
+
+Like links, images also have a footnote style syntax
+
+---
+## 10 Line Break
+
+
+---
```
Comment (ahzamir) [inline src/Components/MarkdoqnPreveiwer.jsx]: - This is too much for a placeholder, please make it short

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
-# MarkdownPreviewer
\ No newline at end of file
+# MarkdownPreviewer 🚀
+
+## Project Description 📝
+
+> This is the Markdown Previewer project created using React it's responsive and a freeCodeCamps frontend library project.
```
Comment (ahzamir) [inline README.md:5]: - Add some description about what your project offers or does

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+
+- I learned how to create a react app.
```
Comment (ahzamir) [inline README.md:18]: - Add what you learned specifically in React while working on this project for example:
  - I learned how to use and create React Components
  - I learned how to use React Hooks and especially `useState` to store the markdown previewer data on it
  - ...

File: README.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+
+
+## Demo 📸
+
+![Demo](/src/images/Demo.png)
+
```
Comment (ahzamir) [inline README.md]: - Include the live demo link

File: README.md
Comment (ahzamir) [inline README.md:1]: - Add the #How To Use section on your project and add the necessary steps like cloning your project, installing dependencies using npm install and running your project on live server using npm start

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Najeeba-Qarqin 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Well done Najeeba, your code passes all the tests and also your codes are clean but still your project design has some issues I pointed out:
  - When the input field is empty it does not have a fixed width so it has some kind of mixing up with the previewer div and does not look great and the opposite with the height, the height of your input field and previewer should be auto so when there is more content there should be more height other than that it should not have fixed height.

image
image

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
❌ Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Markdown Previewer and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Najeeba-Qarqin** [comment]
Hello @ahzamir thanks for your helpful feedback. 
I implemented the feedbacks please check it again! :)

**Review by ahzamir — APPROVED** [review]
### Hi @Najeeba-Qarqin  🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image


# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Markdown Previewer project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**